src/System.Net.ServicePoint/src/System.Net.ServicePoint.builds

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Net.ServicePoint.csproj" />
+    <Project Include="System.Net.ServicePoint.csproj">
```

Comment:
nit: for consistency and future filtering this should have OSGroup=Windows_NT for the net463 build.